the-collab-lab/tcl-20-whats-near-me
# Rf dt placeholder views

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

- Installed `react-router-dom` to handle routing for the 4 views. 
- Added basic, placeholder views with a minimum of styling.
- Map, List, Settings & Help views were created 

## Related Issue
Story: Create placeholder views

## Acceptance Criteria 
- [x] User will be shown the following 4 navigation elements pinned to the bottom of the screen:

- Map
- List
- Settings
- Help

- [x] Tapping on a navigation element will do the following:

- Update the URL of the app
- Map → /
- List → /list
- Settings → /settings
- Help → /help

- [x] Display the unique view associated with each route


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓ | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
| ✓ | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before

### After
![image](https://user-images.githubusercontent.com/59700665/103836338-ceb04480-5056-11eb-8073-0489f974fa41.png)
![image](https://user-images.githubusercontent.com/59700665/103836347-d243cb80-5056-11eb-89cc-7e276eb6da6f.png)
![image](https://user-images.githubusercontent.com/59700665/103836352-d7087f80-5056-11eb-893e-72116522212f.png)
![image](https://user-images.githubusercontent.com/59700665/103836359-dc65ca00-5056-11eb-9edd-a6e29cde58f6.png)

## Testing Steps / QA Criteria

If you click the bottom Nav, you should be able to navigate between the 4 views: map - http://localhost:3000/, list - http://localhost:3000/list, settings - http://localhost:3000/settings & help - http://localhost:3000/help.
